packages/browser/src/sdk.ts

```diff
@@ -266,7 +215,9 @@ function startSessionTracking(): void {
           ?.getSession()
       ) {
         hub.startSession();
-        hub.endSession();
+        // We end the session without removing it from the scope
+        // which is equal to sending it.
+        hub.endSession(false);
```

Comment:
See comments re: `endSession`

```suggestion
  hub.captureSessionUpdate();
```